ts-rs/Cargo.toml

```diff
@@ -55,3 +55,13 @@ thiserror = "1"
 indexmap = { version = "2", optional = true }
 ordered-float = { version = ">= 3, < 5", optional = true }
 serde_json = { version = "1", optional = true }
+
+[profile.test]
+incremental = false
+opt-level = 0
+debug = 0
+lto = false
+debug-assertions = false
+overflow-checks = false
```

Comment:
Was this what gave the major speedups?  
If so, and this has to stay, we definetely need to re-enable debug assertions and overflow checks.

Reply:
This helped, but the main speedup came from putting each test in its own directory

Reply:
> This helped

Nevermind, it seems like profiles are ignored if they aren't in the root Cargo.toml